mozci/push.py

```diff
@@ -534,6 +537,8 @@ def get_shadow_scheduler_tasks(self, name):
         """
         index = self.index + ".source.shadow-scheduler-{}".format(name)
         task = Task(id=find_task_id(index))
+        if task.id==None:
+            return set()
```

Comment:
Do you remember the exception being raised here?

Reply:
This line is propagating the Nones from the `if e.response.status_code == 404:` from `find_task_id()`